doc/classes/Object.xml

```diff
@@ -21,20 +21,21 @@
 			<argument index="0" name="property" type="String">
 			</argument>
 			<description>
-				Return a property, return null if the property does not exist.
+				Returns a property, returns [code]null[/code] if the [code]property[/code] does not exist.
 			</description>
 		</method>
 		<method name="_get_property_list" qualifiers="virtual">
 			<return type="Array">
 			</return>
 			<description>
-				Return the property list, array of dictionaries, dictionaries must contain: name:String, type:int (see TYPE_* enum in [@Global Scope]) and optionally: hint:int (see PROPERTY_HINT_* in [@Global Scope]), hint_string:String, usage:int (see PROPERTY_USAGE_* in [@Global Scope]).
+				Returns this Object's property list as an [Array] of dictionaries, dictionaries must contain: name:String, type:int (see TYPE_* enum in [@Global Scope]) and optionally: hint:int (see PROPERTY_HINT_* in [@Global Scope]), hint_string:String, usage:int (see PROPERTY_USAGE_* in [@Global Scope]).
```

Comment:
- this Object -> the object
- Change comma to period. Second phrase should be a separate sentence.